terraform/app.tf

```diff
@@ -0,0 +1,99 @@
+resource "azurerm_service_plan" "service-plan" {
+  name                = "getanid-authserver-${var.environment_name}-app-spln"
+  location            = data.azurerm_resource_group.group.location
+  resource_group_name = data.azurerm_resource_group.group.name
+  os_type             = "Linux"
+  sku_name            = var.app_service_plan_sku
+}
+
+resource "azurerm_linux_web_app" "auth-server-app" {
+  name                = local.get_an_identity_app_name
+  location            = data.azurerm_resource_group.group.location
+  resource_group_name = data.azurerm_resource_group.group.name
+  service_plan_id     = azurerm_service_plan.service-plan.id
+  https_only          = true
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  site_config {
+    http2_enabled       = true
+    minimum_tls_version = "1.2"
+  }
+
+  app_settings = {
+    HOSTING_ENVIRONMENT                    = var.environment_name,
+    APPLICATION_INSIGHTS_CONNECTION_STRING = azurerm_application_insights.insights.connection_string
+    REDIS_URL                              = "${azurerm_redis_cache.redis.hostname}/${azurerm_redis_cache.redis.primary_access_key}",
+    AZURE_STORAGE_ACCOUNT_NAME             = azurerm_storage_account.data-protection.name,
+    AZURE_STORAGE_ACCESS_KEY               = azurerm_storage_account.data-protection.primary_access_key,
+    AZURE_STORAGE_CONTAINER                = azurerm_storage_container.uploads.name
+  }
```

Comment:
Can we get the Postgres connection string in here too?

Reply:
Any specific config key name that you want for  postgres connectionstring ?

Reply:
> Any specific config key name that you want for postgres connectionstring ?

Ideally `ConnectionStrings__DefaultConnection`